### ISSUE TITLE
Intel version gobbling mem

### ISSUE THREAD

**Issue by cori** [jasonjmcghee/rem#77]
I've been running the intel version on and off on an intermittently-used 2019 16" MBP. I hadn't noted this until recently (so perhaps related to a recent Sonoma developer beta?) but it's been gobbling memory and causing my MBP to crash.

I'd recently recovered from a crash and hadn't yet restarted rem but had activity monitor up and was doing some "normal computing". I started rem and started recording, piddled around a bit and wandered away for a few minutes (maybe 10?). the WindowsServer memoruy was at ~1.5Gb and when i came back was almost at 10Gb. i just grabbed a rem video from a little while before while the laptop was sitting mostly idle and watched WindowsServer grow in memory from 4-5 Gb in the span of a single mp4.

I'm going to stop running it here at least until the next beta update and see how things look then; perhaps others not running the sonoma beta are havign better luck.

**Comment by jasonjmcghee** [comment]
Wow crazy. Thanks for logging.

Pinning this- serious issue. 

Very curious if anyone has any insight as to why this might be happening.

Are we improperly handling a manager that is leaking memory?

**Comment by cori** [comment]
I'll play around a bit more - I _thought_ I'd uncovered some evidence that `rem` had been unrelated - running it with nothing else going in both recording and non-recording states for a while without memory impact - but it turns out `rem` had stopped, so ¯\_(ツ)_/¯.

**Comment by cori** [comment]
ha ok wow that was _definitely_ not what I saw.

on the intel mac, running "nothing" but activity monitor and `rem` (I guess HomeAssistant was running in the bg - it popped up a window at some point), once i turned on Recording the WindowServer memory immediately started to climb, and reach 19Gb within around 15m.

I grabbed Activity Monitor samples from both WindowServer and rem that I can upload somwhere if they're helpful - no idea how anonymous / secure those are tho. not sure if that's the sort of thing that would help troubleshoot this or not.

For the time being I'm not going to run rem over there until the next Sonoma Beta update as i mentioned, but happy to hellp in any way i can
